examples/images_contours_and_fields/triinterp_demo.py

```diff
@@ -29,33 +29,31 @@
 interp_cubic_min_E = mtri.CubicTriInterpolator(triang, z, kind='min_E')
 zi_cubic_min_E = interp_cubic_min_E(xi, yi)
 
+# Set up the figure
+fig, axes = plt.subplots(nrows=2, ncols=2)
```

Comment:
Not a fan of `axes` becuase of the plural/singular ambiguity....

Reply:
fair point

Reply:
super minor - I'd only change if someone else has serious issues...